app/assets/javascripts/govuk_publishing_components/components/feedback.js

```diff
@@ -20,6 +20,7 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
       this.$promptQuestions = $element.find('.js-prompt-questions')
       this.$promptSuccessMessage = $element.find('.js-prompt-success')
       this.$somethingIsWrongForm = $element.find('#something-is-wrong')
+      this.$pageIsNotUsefulForm = $element.find('#page-is-not-useful')
```

Comment:
This doesn't sound very promising about our forms! Perhaps `this.$surveyForm` is better?

Reply:
👍